aio/content/marketing/events.json

```
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "ng-conf",
+    "location": "Online",
+    "linkUrl": "https://www.2021.ng-conf.org/",
+    "date": {
+      "start": "2021-04-22",
+      "end": "2021-04-23"
+    }
```

Note that you can add another entry for workshops if desired:

```suggestion
    },
    "workshopsDate": {
      "start": "2021-04-12",
      "end": "2021-04-15"
    }
```